### ISSUE TITLE
BART fine-tuning script for XSum dataset

### ISSUE THREAD

**Issue by vprelovac** [pytorch/fairseq#1664]
## 🚀 Feature Request

Please include fine-tuning scipt you used to produce XSum dataset results. 

### Motivation

I've used modified CNN/Dailiy Mail training script for XSum (on 8xV100 32GB) but it achieves about 10% worse results than those published in the paper.

### Pitch

Results reproducability :)

**Comment by ngoyal2707** [comment]
@yinhanliu

**Comment by YizhuLiu** [comment]
@vprelovac Have you evaluated the bart.large.cnn model? Did you get the same R-2 score on CNN/DM datase as published? I used pre-trained model to fine-tune CNN/DM training. But the ROUGE-2 is 19.19 (R-2 in published paper is 21.28).

**Comment by vprelovac** [comment]
@YizhuLiu No we havent tried training it on CNN/DM.

On Xsum we used same script just modified lenpen from 1 to 2  on output and got these results.

ROUGE-1 Average_F: 0.44489 (95%-conf.int. 0.44198 - 0.44752)
1 ROUGE-2 Average_F: 0.21043 (95%-conf.int. 0.20758 - 0.21324)
1 ROUGE-L Average_F: 0.35347 (95%-conf.int. 0.35062 - 0.35628)

These are results from paper:
ROUGE-1: 45.14
ROUGE-2: 22.27
ROUGE-L: 37.25

**Comment by pltrdy** [comment]
@vprelovac do you

**Comment by vprelovac** [comment]
@pltrdy using their included script. But rouge is very inadequate as a metric for summarization tasks anyway as it does not account for factual inaccuracies. In my experience with BART I can say that it produces adequate summaries 80%-90% of time (which is fantastic) and most bad ones are factual inaccuracies or 'daydreaming'.

**Comment by zsquaredz** [comment]
The README.md has been updated under examples/bart with hyper-parameters used for XSum dataset, and the trained model is also released. 
https://github.com/pytorch/fairseq/blob/master/examples/bart/README.md
